sites/fast-component-explorer/package.json

```diff
@@ -12,7 +12,7 @@
     "prettier": "prettier --config ../../.prettierrc --write \"**/*.{ts,tsx,html}\"",
     "prettier:diff": "prettier --config ../../.prettierrc \"**/*.{ts,tsx,html}\" --list-different",
     "start": "webpack-dev-server --config=./webpack.dev.js --history-api-fallback --progress",
-    "test": "yarn convert:readme && yarn eslint && yarn build:app",
+    "test": "yarn eslint",
```

Comment:
I don't think the `build:app` step will be run during any existing part of our build process. Is it possible that an update to components causes a break (API change, etc) which would then not be caught by ensuring that the app builds properly. A big part of running this during the test step was because the app building in itself is *the test*. How do we address that scenario with this change?